app/src/main/java/io/intrepid/contest/screens/sendinvitations/selectcontacts/SelectContactsAdapter.java

```diff
@@ -4,15 +4,15 @@
 import android.support.v7.widget.RecyclerView;
 import android.view.ViewGroup;
 
-import java.util.ArrayList;
+import java.util.List;
 
 import io.intrepid.contest.models.Contact;
 
 public class SelectContactsAdapter extends RecyclerView.Adapter<SelectContactsViewHolder> {
 
-    private ArrayList<Contact> contactsList;
+    private List<Contact> contactsList;
```

Comment:
i think i mentioned this in the other PR but make this final and instead of assigning the list, call `clear` and `addAll` on `contactsList` to update it